.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 orbs:
   win: circleci/windows@2.4.0
   percy: percy/agent@0.1.3
-  browser-tools: circleci/browser-tools@1.4.3
+  browser-tools: circleci/browser-tools@1.4.5
```

Comment:
There's a 1.4.6 now, but we can wait on that if this one works.

Reply:
oh maybe that would let us remove the chrome 116 pin?

Reply:
Yes I think that was fixed and there is also now a proper chromedriver version I think that was the cause for dvpc001 failures.

Reply:
I unpinned the chrome version and the dvpc was failing again.

Reply:
OK - please make an issue to come back to this